src/compilation/compiler.rs

```diff
@@ -355,123 +352,87 @@ impl Compiler {
         LLVMPositionBuilderAtEnd(self.builder, after_block);
     }
 
-    /// Returns the function currently being built by the compiler.
-    unsafe fn get_cur_function(&self) -> LLVMValueRef {
-        let cur_func = LLVMGetBasicBlockParent(LLVMGetInsertBlock(self.builder));
-        if cur_func.is_null() {
-            panic!("Builder is not inside a function; can't get current function.");
-        }
-        cur_func
-    }
-
-    /// Compiles 0 or more variable declarations.
-    unsafe fn compile_var_declarations(&mut self, var_declarations: &[VarDeclaration]) {
-        let func = self.get_cur_function();
-        for var_declaration in var_declarations {
-            let var_name = var_declaration.var_name.as_str();
-            let var_type = var_declaration.var_type;
-            let alloca = self.create_entry_block_alloca(func, var_name, var_type);
-            self.scope_manager.set_var(var_name, var_type, alloca);
-
-            if let Some(value_expr) = &var_declaration.var_value {
-                let value = self.compile_expr(value_expr, var_type);
-                LLVMBuildStore(self.builder, value, alloca);
-            }
-        }
-    }
-
     /// Compiles an assignment expression like `foo = 28` (and panics if `foo`'s type can't store 28).
```

Comment:
Just to leave a note to ourselves when we start handling error messages
```suggestion
    /// Compiles an assignment expression like `foo = 28` (and panics if `foo`'s type can't store 28).
    // TODO: remove the panics from here and make sure they're in Typer
```